src/magic_folder/test/test_web.py

```diff
@@ -246,7 +267,7 @@ def authorized_request(treq, auth_token, method, url):
     )
 
 
-def treq_for_folders(basedir, auth_token, folders):
+def treq_for_folders(reactor, basedir, auth_token, folders, start_folder_services):
```

Comment:
"reactor" is pretty obvious, but I guess it should be added to the docstring too..